sdk/springappdiscovery/Azure.ResourceManager.SpringAppDiscovery/tests/Tests/AppCRUDTests.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.SpringAppDiscovery.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.SpringAppDiscovery.Tests.Tests
+{
+    [TestFixture]
+    public class AppCRUDTests : SpringAppDiscoveryManagementTestBase
+    {
+        public AppCRUDTests() : base(true)
+        {
+        }
+
+        [SetUp]
+        public async Task ClearAndInitialize()
+        {
+            if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
+            {
+                await CreateCommonClient();
+            }
+        }
+
+        /// <summary>
+        /// Test App CRUD for spring discovery
+        /// </summary>
+        /// <returns></returns>
+        [TestCase]
```

Comment:
Let us use the `RecordedTest` attribute instead.
This is an attribute introduced by our test framework which inherits from TestCase

Reply:
Accroding to the test guide:[https://github.com/Azure/azure-sdk-for-net/blob/3e5807cba6bc6d7d417ef722ff2946b42f2412cb/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md](url), the demo shows should add both TestCaseand RecordedTest, and I use both of them and it worked. If I just use RecordedTest, the test case seems to run in recording mode, so I keep the  TestCase.